bodhi/server/validators.py

```diff
@@ -800,48 +778,60 @@ def validate_update(request, **kwargs):
         request.errors.status = HTTPNotFound.code
 
 
-def validate_update_owner(request, **kwargs):
+def common_validate(param, request):
     """
-    Make sure the referenced update owner is an existing user.
+    Common fuction to be called for validatation.
```

Comment:
Let's specify here that this is specifically for ensuring the referenced user exists. Perhaps we could say "Ensure the user referenced by param exists, and if it does replace it with the User object."""